src/fusor/harvester.py

```diff
@@ -29,6 +29,19 @@ class FusionCallerHarvester(ABC):
     column_rename: dict
     delimeter: str
 
+    def _get_records(self, fusions_file: TextIO) -> csv.DictReader:
+        """Read in all records from a fusions file
+
+        :param fusion_path: The open fusions file
+        :return A csv.DictReader object containing the detected fusions
+        """
+        if self.fusion_caller is EnFusion:
+            fusion_lines = dropwhile(
+                lambda line: not line.startswith("UnorderedFusion"), fusions_file
+            )
+            return csv.DictReader(fusion_lines, delimiter=self.delimeter)
+        return csv.DictReader(fusions_file, delimiter=self.delimeter)
```

Comment:
I was thinking of something like this for the base class

```suggestion
    def _get_records(self, fusions_file: TextIO) -> csv.DictReader:
        """Read in all records from a fusions file

        :param fusion_path: The open fusions file
        :return A csv.DictReader object containing the detected fusions
        """
        return csv.DictReader(fusions_file, delimiter=self.delimeter)
```

and then in the EnFusion harvester:

```
        def _get_records(self, fusions_file: TextIO) -> csv.DictReader:
            """Read in all records from a fusions file
    
            :param fusion_path: The open fusions file
            :return A csv.DictReader object containing the detected fusions
            """
            fusion_lines = dropwhile(
                lambda line: not line.startswith("UnorderedFusion"), fusions_file
            )
            return csv.DictReader(fusion_lines, delimiter=self.delimeter)
```